### PR TITLE
[Skills Everywhere][#259] Add SignIn test

### PR DESCRIPTION
**Pull request by denscollo**
Addresses #259

## Description

This PR adds the Sign In tests to the SkillFunctionalTests project.

### Specific Changes

- Updated SignInTest.cs file to run the script to the point to get the SignIn URL, authenticate and continue the dialog with the bot.
- Added the scripts for the dialog with the SignIn bot split into two scripts, SignIn1 and SignIn2, the first with the dialog up to the SignIn request and the last with the rest after the authentication.

### CODE REVIEW

**Comment by denscollo** [comment]
@gabog we replaced the files with the ones that you sent us and tested it. We just pushed the changes. Thanks!

File: Bots/DotNet/Skills/CodeFirst/WaterfallSkillBot/Dialogs/Auth/AuthDialog.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public AuthDialog(IConfiguration configuration)
         {
             _connectionName = configuration["ConnectionName"];
 
+            AddDialog(new ConfirmPrompt(nameof(ConfirmPrompt)));
```
Comment (gabog) [inline Bots/DotNet/Skills/CodeFirst/WaterfallSkillBot/Dialogs/Auth/AuthDialog.cs:22]:
```suggestion
            // This confirmation dialog should be removed once https://github.com/microsoft/BotFramework-FunctionalTests/issues/299 is resolved (and this class should look like the class in the issue)
            AddDialog(new ConfirmPrompt(nameof(ConfirmPrompt)));
```

**Review by gabog — CHANGES_REQUESTED** [review]
Thanks @denscollo, I just added a comment that points to the issue I created so we remember to remove this code once we fix the SDK.
Make the PR non-draft If everything is passing in your pipelines and I will approve and merge into main

**Comment by denscollo** [comment]
@gabog thanks! The PR is ready to review now.

**Review by gabog — APPROVED** [review]

**Pull request merged**